x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/components/create_analytics_form/create_analytics_form.tsx

```diff
@@ -80,14 +76,44 @@ export const CreateAnalyticsForm: FC<CreateAnalyticsFormProps> = ({ actions, sta
     sourceIndex,
     sourceIndexNameEmpty,
     sourceIndexNameValid,
+    sourceIndexContainsNumericalFields,
+    sourceIndexFieldsCheckFailed,
     trainingPercent,
   } = form;
 
+  // Find out if index pattern contain numeric fields. Provides a hint in the form
+  // that an analytics jobs is not able to identify outliers if there are no numeric fields present.
+  const validateSourceIndexFields = async () => {
+    try {
+      const indexPattern: IndexPattern = await kibanaContext.indexPatterns.get(
```

Comment:
I should have spotted this before, but the full list of index patterns has already been loaded in the routing resolver by the call to `loadIndexPatterns`.
This means you use one of the util functions available in `index_utils.ts`.
I don't think the index patterns being cached by `loadIndexPatterns` is being used at all in this page and so if you choose not to use it, it could possibly be removed from the resolver.

Reply:
Good catch - removed from the resolver 👌